sample/src/main/java/com/karumi/dexter/sample/PermissionsViews.java

```diff
@@ -0,0 +1,49 @@
+package com.karumi.dexter.sample;
+
+import android.Manifest;
+import android.widget.TextView;
+import butterknife.BindView;
+
+public class PermissionsViews {
+    @BindView(R.id.audio_permission_feedback)
+    TextView audioPermissionFeedbackView;
+    @BindView(R.id.camera_permission_feedback)
+    TextView cameraPermissionFeedbackView;
+    @BindView(R.id.contacts_permission_feedback)
+    TextView contactsPermissionFeedbackView;
+
+    PermissionsViews(TextView audioView, TextView cameraView, TextView contactView) {
```

Comment:
I'm seeing you are receiving the views in the constructor but we are only calling it with null values so it will crash with an NPE the moment you use any of them.